tests/fake/test_fake_publish.py

```diff
@@ -63,3 +73,52 @@ def test_publish_absent_raises():
     exception = repo_copy2.publish().exception()
     assert isinstance(exception, PulpException)
     assert "repo1 not found" in str(exception)
+
+
+def test_can_upload(tmpdir):
```

Comment:
Please don't add the upload test to the file named test_fake_publish, as publish and upload are different things.  In this project, I tried to keep test files smaller and more specific so we don't end up with what we see in many other projects, where each test file keeps growing indefinitely and it's tough to clean them up or to understand the relationship between tests and fixtures.

Can you please add that to a new file, test_fake_upload.py?  There's nothing wrong with having a test file starting with only 1-2 tests.

Reply:
whoops.